ti/instrumentation/java/helper.go

```diff
@@ -46,6 +47,8 @@ const (
 	JAVA_SRC_PATH      = "src/main/java/"      //nolint:revive,stylecheck
 	JAVA_TEST_PATH     = "src/test/java/"      //nolint:revive,stylecheck
 	JAVA_RESOURCE_PATH = "src/test/resources/" //nolint:revive,stylecheck
+	SCALA_TEST_PATH    = "src/test/scala/"     //nolint:revive,stylecheck
```

Comment:
Do we basically classify a test type based on if it is present at this location or no? Can we come up with something more robust?

Reply:
without static code analysis, aka reading the content of the code. No